source/basic.tex

```diff
@@ -1752,6 +1752,32 @@
 If that lookup finds nothing, it undergoes unqualified name lookup;
 in each case, only names
 that denote types or templates whose specializations are types are considered.
+\begin{example}
+\begin{codeblock}
+struct T1 { struct U { int i; }; };
```

Comment:
`T1::U` can (now?) be omitted entirely.

Reply:
Well, maybe someone believes we'll look up U inside T, so I want to leave this in.

Reply:
OK; no further questions.